roles/sushy_emulator/tasks/render_resources.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Generate htpasswd string
+  ansible.builtin.command:
```

Comment:
apparently there's a module for that:
https://docs.ansible.com/ansible/latest/collections/community/general/htpasswd_module.html
Might be nice to call? It seems to write the result in a file directly (didn't see for now where you're using that string, probably in the template bellow?)

Reply:
Found it: it's indeed in the template, and leveraging the module wouldn't make any sense.

BTW are you sure this `htpasswd` binary is installed on the system? :)